src/basic/log.c

```diff
@@ -405,7 +405,7 @@ static int write_to_console(
                 const char *lon = "", *loff = "";
                 if (log_get_show_color()) {
                         lon = ansi_highlight_yellow4();
-                        loff = ANSI_NORMAL;
+                        loff = ansi_normal();
```

Comment:
btw, the reason this used ANSI_NORMAL was that this code already checked whether color is enabled, and ansi_normal() just checks this again